src/main/java/me/clip/placeholderapi/commands/PlaceholderAPICommands.java

```diff
@@ -110,7 +110,7 @@ public boolean onCommand(CommandSender s, Command c, String label, String[] args
 				return eCloud.onCommand(s, c, label, args);
 				
 			} else if (args[0].equalsIgnoreCase("enablecloud")) {
-				if (!s.isOp()) {
+				if (!s.hasPermission("placeholderapi.ecloud")) {
```

Comment:
Should the permission be specific? `placeholderapi.ecloud.toggle` or something.

Reply:
I thought of doing that, but to be honest, if you have enough power to install extensions on the server, you most likely have full rights (or close to full rights)